### DISCUSSION TITLE
firefox 38.7 policy

### DISCUSSION THREAD

**Discussion by ektrontek** [mozilla/policy-templates#745] [Q&A] [answered]
Hi ,
Would it be possible to find firefox38.7 policy-templates to use? Thanks a lot

**Comment by mkaply** [comment, marked as answer]
We added policies in Firefox 60.

Firefox 38.7 is 5 years old. You shouldn't be using a version of Firefox that old.

**Reply by ektrontek** [reply]
Thanks a lot. Would it be possible to change firefox privacy setting by regedit setting
![2021_03_18_19_13_52_Options](https://user-images.githubusercontent.com/19470763/111641723-5fb25180-8838-11eb-8042-75dee407d5ee.jpg)

**Reply by mkaply** [reply]
There are no registry settings to configure that old of a version of Firefox.

You would have to use the CCK2 tool to configure all of that.

**Comment by ektrontek** [comment]
Thanks a lot. I have CCK2 tool, ex:cck2wizard-latest.xpi, but I don't know
how to use it to deploy other computer setting.

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
不含病毒。www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by mkaply** [comment]
The CCK2 tool generates a ZIP file that you unzip into your Firefox install.

https://mike.kaply.com/2015/01/13/cck2-what-now/

**Comment by ektrontek** [comment]
I found one information : Integrating Add-ons into Firefox | Mike's Musings
(kaply.com)
<https://mike.kaply.com/2012/02/09/integrating-add-ons-into-firefox/>
-->
Not available as of Firefox 40
Does it mean firefox38.7 can not be use for CCK2 tool.

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
不含病毒。www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by mkaply** [comment]
You can use the tool with 38.7. I think We should take this conversation to email. Are you the same person that posted to my blog with questions? If so, I'll email you directly.

**Comment by ektrontek** [comment]
Yes,thanks for help.

Michael Kaply ***@***.***> 於 2021年3月20日 週六 上午1:12 寫道：

**Comment by ektrontek** [comment]
Hi, After I try , cki can not install in fiefox38.7, how can I adjust?


<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
不含病毒。www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by ektrontek** [comment]
Hi,
I have unstandstand how to setup mozila.cfg, but I don't know how to add a
exception for firefox, would it be possible to guidance about this? I try
below, but it fail to work.Thanks a lot.
Regards, Hsu Yao Chang


<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
不含病毒。www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
